benchmarks/traversals.hs

```diff
@@ -20,27 +20,6 @@ import Control.Lens
 import Criterion.Main
 import Criterion.Types
 
-#if !(MIN_VERSION_bytestring(0,10,0))
```

Comment:
Now that this code is gone, we should be able to remove the `{-# LANGUAGE CPP #-}` and `{-# OPTIONS_GHC -fno-warn-orphans #-}` lines above.